sdcm/mgmt.py

```diff
@@ -243,6 +243,15 @@ def detailed_progress(self):
         relevant_key = [key for key in parsed_progress_table.keys() if parsed_progress_table[key]][0]
         return parsed_progress_table[relevant_key]
 
+    def wait_for_task_success_status(self, timeout=10800):
```

Comment:
perhaps rename to "wait_for_task_done_status"

Reply:
+1